code/core/src/core-server/utils/server-statics.ts

```diff
@@ -60,17 +60,19 @@ export async function useStatics(app: Polka, options: Options): Promise<void> {
 
   // Fix for serving favicon in dev mode - use the directory containing the favicon
   // rather than trying to serve the file directly
-  const faviconDir = resolve(faviconPath, '..');
-  const faviconFile = basename(faviconPath);
-  app.use('/', (req, res, next) => {
-    if (req.url === `/${faviconFile}`) {
-      return sirvWorkaround(faviconDir, {
-        dev: true,
-        etag: true,
-        extensions: [],
-      })(req, res, next);
-    }
-    next();
+  faviconPaths.forEach((faviconPath) => {
+    const faviconDir = resolve(faviconPath, '..');
+    const faviconFile = basename(faviconPath);
+    app.use('/', (req, res, next) => {
+      if (req.url === `/${faviconFile}`) {
+        return sirvWorkaround(faviconDir, {
+          dev: true,
+          etag: true,
+          extensions: [],
+        })(req, res, next);
+      }
+      next();
+    });
```

Comment:
maybe it would be better if we could do this loop _within_ the middleware. So instead of creating 6+ middlewares (`app.use()` calls), we'd still only have one, and then the loop within it would return the correct thing.

Maybe:

```suggestion
  app.use('/', (req, res, next) => {
    for (const faviconPath of faviconPaths) {
      const faviconDir = resolve(faviconPath, '..');
      const faviconFile = basename(faviconPath);
      if (req.url === `/${faviconFile}`) {
        return sirvWorkaround(faviconDir, {
          dev: true,
          etag: true,
          extensions: [],
        })(req, res, next);
      }
    }
    next();
  });
```